cloud_native_security.md

```diff
@@ -0,0 +1,37 @@
+# CLOUD NATIVE SECURITY DAY
+One of the things we do as a community is a Cloud Native Security day, it is designed to bring together the cloud native security community together to discuss and share current challenges and solutions in cloud native security to get together in a vendor neutral space.
+
+Cloud Native Security Day (#cnsecurityday) is intended to drive collaboration, discussion, and knowledge sharing of cloud native security accomplishments and roadblocks. Get connected with others that are passionate about security. Learn from practitioners about pitfalls to avoid, hurdles to jump, and how to integrate security into your cloud native project, architecture, and enhance team awareness on security.
+
+# Here are some references to some of the past events & some of the issues that detail the organization/planning of these events:
+
+## 2016
+
+KubeCon + CloudNativeCon North America, Seattle, USA - November 8-9 2016
+[CloudNativeCon + Security](https://cnkc16.sched.com/overview/type/CloudNativeCon+Security) - November 9, 2016
+
+## 2019
+
+[Cloud Native Security Day, San Diego, USA](https://events19.linuxfoundation.org/events/cloud-native-security-day-2019/) - November 18, 2019
+[issue#209](https://github.com/cncf/tag-security/issues/209) - Planning/Organization of the event
+
+
+## 2020
+
+[Cloud Native Security Day, Virtual](https://events.linuxfoundation.org/archive/2020/cloud-native-security-day/) - August 17, 2020
+
+
+[Cloud Native Security Day North America, Virtual](https://events.linuxfoundation.org/cloud-native-security-day-north-america/) - November 17, 2020
+[issue#416](https://github.com/cncf/tag-security/issues/416)
```

Comment:
```suggestion
[Issue #416](https://github.com/cncf/tag-security/issues/416)
```

Let's also maybe put these as bullet points, which will help formatting be a bit easier.

- Event 1
  - Issue link 
- Event 2
  - Issue link